### PR TITLE
updating projects page with restriction rules

### PR DESCRIPTION
**Pull request by emuus**

### CODE REVIEW

File: admin/en/source/pages/manage-resources/manage-project-catalog.rst [hunk 1]
```diff
@@ -41,34 +39,34 @@ To add a new project:
...
 
-	6. Optionally, you can add the following meta-data on the ``Overview`` tab:
+	5. Optionally, you can add the following meta-data on the ``Overview`` tab:
 	
 		* a logo for the project
 		* website information of where the project came from
 		* description
 
-	7. You can (optionally) restrict the distributions that the project applies to. On the ``Restrictions`` page select the distribution from the bottom list and click the upward arrow. Click ``save``.
+	6. You can (optionally) restrict the distribution or target format that the project applies to, from the ``Restrictions`` page. The restriction rule is set as a regular expression. For more information, refer to :ref:``. Click ``save``.
```
Comment (lqueiroga) [inline admin/en/source/pages/manage-resources/manage-project-catalog.rst]: Is it normal that there is no page at the end when you said "refer to :ref" ?

File: admin/en/source/pages/manage-resources/manage-project-catalog.rst [hunk 2]
```diff
@@ -77,7 +75,31 @@ To add a new project:
...
+	3. Go to the ``Restrictions`` tab. Enter the restriction rule. The restriction rule is represented by a logical expression with format ``Object#field=value`` or ``Object#field!=value``, where:
+		* ``object`` is either ``Distribution`` or ``TargetFormat``
+		* for ``Distribution`` field is ``family``, ``pkgType``, ``name``, ``version`` or ``arch``. The ``version`` must be a major version.
+		* for ``TargetFormat`` field is ``name`` or ``type`` 
+		* ``value`` is the value you want to match with the fields. For example, ``CentOS`` for Distribution name, ``linux`` for Distribution family, ``x86_64`` for Distribution arch, ``VirtualBox`` for TargetFormat name, ``cloud`` for TargetFormat type.
+		* logical operator is ``||`` for OR and ``&&`` for AND
+
+For example, if the software bundle is designed only for distributions CentOS 7 x86_64 or Debian 8 x86_64, or for TargetFormat with type virtual, then you would note the Restriction rule as follows:: 
+
+	(Distribution#arch=x86_64 && ((Distribution#name=CentOS && Distribution#version=7) || (Distribution#name=Debian && Distribution#version=8)) || TargetFormat#type=virtual
```
Comment (lqueiroga) [inline admin/en/source/pages/manage-resources/manage-project-catalog.rst]: Missing parenthesis before "|| TargetFormat#type=virtual" (there is a commit to fix it in UI). The good one is:
(Distribution#arch=x86_64 && ((Distribution#name=CentOS && Distribution#version=7) || (Distribution#name=Debian && Distribution#version=8))) || TargetFormat#type=virtual

Comment (emuus) [inline admin/en/source/pages/manage-resources/manage-project-catalog.rst]: ok. copied your example

File: admin/en/source/pages/manage-resources/manage-project-catalog.rst (hunk 1 above)
Comment (emuus) [inline admin/en/source/pages/manage-resources/manage-project-catalog.rst]: will be added in next PR

File: end-user/en/source/pages/templating/appliance-software.rst [hunk 3]
```diff
@@ -103,20 +103,52 @@ To add custom software components to an appliance:
...
+	16. Select the software components you want to add and click the down arrow button. 
+	17. Click ``save`` to add this software component to your appliance template.
+
+.. _restriction-rules:
+
+Restricting Projects for Software Bundles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+UForge allows you to restrict projects for the files you add under ``My Software``.
+
```
Comment (lqueiroga) [inline end-user/en/source/pages/templating/appliance-software.rst]: Title and sentence are not clear: 'Restricting Software components for OSes and Formats.

File: admin/en/source/pages/manage-resources/manage-project-catalog.rst [hunk 4]
```diff
@@ -77,7 +75,31 @@ To add a new project:
 
 		.. image:: /images/create-project-bootscript.png
 	
-	11. Click ``Save``.
+	10. Click ``Save``.
+
+.. _restrict-distrib:
+
+Restricting Projects for Software Bundles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (lqueiroga) [inline admin/en/source/pages/manage-resources/manage-project-catalog.rst]: Title is not clear: 'Restricting Projectss for OSes and Formats ?

File: admin/en/source/pages/manage-resources/manage-project-catalog.rst [hunk 5]
```diff
@@ -77,7 +75,31 @@ To add a new project:
...
+
+	1. Under the ``Administration`` tab, click ``Projects``.
+	2. Select the project you want to modify.
+	3. Go to the ``Restrictions`` tab. Enter the restriction rule. The restriction rule is represented by a logical expression with format ``Object#field=value`` or ``Object#field!=value``, where:
+		* ``object`` is either ``Distribution`` or ``TargetFormat``
+		* for ``Distribution`` field is ``family``, ``pkgType``, ``name``, ``version`` or ``arch``. The ``version`` must be a major version.
+		* for ``TargetFormat`` field is ``name`` or ``type`` 
+		* ``value`` is the value you want to match with the fields. For example, ``CentOS`` for Distribution name, ``linux`` for Distribution family, ``x86_64`` for Distribution arch, ``VirtualBox`` for TargetFormat name, ``cloud`` for TargetFormat type.
+		* logical operator is ``||`` for OR and ``&&`` for AND
+
```
Comment (lqueiroga) [inline admin/en/source/pages/manage-resources/manage-project-catalog.rst]: In the UI we added a new instruction point:
* Carriage return is not authorized

File: end-user/en/source/pages/templating/appliance-software.rst [hunk 6]
```diff
@@ -103,20 +103,52 @@ To add custom software components to an appliance:
...
+	1. Under the ``Apps`` go to the ``Software Library`` page.
+	2. Select the sofware you want to modify.
+	3. Go to the ``Restrictions`` tab. Enter the restriction rule. The restriction rule is represented by a logical expression with format ``Object#field=value`` or ``Object#field!=value``, where:
+		* ``object`` is either ``Distribution`` or ``TargetFormat``
+		* for ``Distribution`` field is ``family``, ``pkgType``, ``name``, ``version`` or ``arch``. The ``version`` must be a major version.
+		* for ``TargetFormat`` field is ``name`` or ``type`` 
+		* ``value`` is the value you want to match with the fields. For example, ``CentOS`` for Distribution name, ``linux`` for Distribution family, ``x86_64`` for Distribution arch, ``VirtualBox`` for TargetFormat name, ``cloud`` for TargetFormat type.
+		* logical operator is ``||`` for OR and ``&&`` for AND
+
```
Comment (lqueiroga) [inline end-user/en/source/pages/templating/appliance-software.rst]: In the UI we added a new instruction point:
* Carriage return is not authorized

File: end-user/en/source/pages/templating/appliance-software.rst (hunk 3 above)
Comment (emuus) [inline end-user/en/source/pages/templating/appliance-software.rst]: ok

File: end-user/en/source/pages/templating/appliance-software.rst (hunk 6 above)
Comment (emuus) [inline end-user/en/source/pages/templating/appliance-software.rst]: ok added

File: admin/en/source/pages/manage-resources/manage-project-catalog.rst (hunk 5 above)
Comment (emuus) [inline admin/en/source/pages/manage-resources/manage-project-catalog.rst]: ok added

File: admin/en/source/pages/manage-resources/manage-project-catalog.rst (hunk 4 above)
Comment (emuus) [inline admin/en/source/pages/manage-resources/manage-project-catalog.rst]: ok

**Pull request merged**